src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedPrivateImplementationDetailsType.cs

```diff
@@ -0,0 +1,189 @@
+
+        public override string Name => _privateImplementationDetails.Name!;
+
+        public override IEnumerable<string> MemberNames => SpecializedCollections.EmptyEnumerable<string>();
+
+        public override Accessibility DeclaredAccessibility => Accessibility.Internal;
+
+        public override bool IsSerializable => false;
+
+        public override bool AreLocalsZeroed => false;
+
+        public override TypeKind TypeKind => TypeKind.Class;
+
+        public override bool IsRefLikeType => false;
+
+        public override bool IsReadOnly => false;
+
+        public override Symbol ContainingSymbol => _globalNamespace;
+
+        public override ImmutableArray<Location> Locations => ImmutableArray<Location>.Empty;
+
+        public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences => ImmutableArray<SyntaxReference>.Empty;
+
+        public override bool IsStatic => _privateImplementationDetails.IsSealed && _privateImplementationDetails.IsAbstract;
+
+        public override bool IsAbstract => _privateImplementationDetails.IsAbstract && !_privateImplementationDetails.IsSealed;
+
+        public override bool IsSealed => _privateImplementationDetails.IsSealed && !_privateImplementationDetails.IsAbstract;
+
+        internal override ImmutableArray<TypeWithAnnotations> TypeArgumentsWithAnnotationsNoUseSiteDiagnostics => ImmutableArray<TypeWithAnnotations>.Empty;
+
+        internal override bool IsFileLocal => false;
+
+        internal override FileIdentifier? AssociatedFileIdentifier => null;
+
+        internal override bool MangleName => false;
+
+        internal override bool HasDeclaredRequiredMembers => false;
+
+        internal override bool HasCodeAnalysisEmbeddedAttribute => false;
+
+        internal override bool IsInterpolatedStringHandlerType => false;
+
+        internal override bool HasSpecialName => _privateImplementationDetails.IsSpecialName;
+
+        internal override bool IsComImport => false;
+
+        internal override bool IsWindowsRuntimeImport => false;
+
+        internal override bool ShouldAddWinRTMembers => false;
+
+        internal override TypeLayout Layout => new TypeLayout(_privateImplementationDetails.Layout, (int)_privateImplementationDetails.SizeOf, (byte)_privateImplementationDetails.Alignment);
+
+        internal override CharSet MarshallingCharSet => _privateImplementationDetails.StringFormat;
+
+        internal override bool HasDeclarativeSecurity => false;
+
+        internal override bool IsInterface => false;
+
+        internal override NamedTypeSymbol? NativeIntegerUnderlyingType => null;
+
+        internal override NamedTypeSymbol BaseTypeNoUseSiteDiagnostics => _objectType;
+
+        internal override bool IsRecord => false;
+
+        internal override bool IsRecordStruct => false;
+
+        internal override ObsoleteAttributeData? ObsoleteAttributeData => null;
+
+        public override ImmutableArray<Symbol> GetMembers() => ImmutableArray<Symbol>.Empty;
```

Comment:
>ImmutableArray<Symbol>.Empty [](http://example.com/codeflow?start=63&length=28)

Is this correct?

Reply:
> Is this correct? 

Yes, this is correct. There is no expectation that this API returns synthesized members (there is nothing new in this behavior for symbols) and all members of this type are synthesized.